### PR TITLE
doc(README.md): Modify the README.md content according to the project

### PR DESCRIPTION
**Pull request by SachinthaSampath**
# Standardize README.md

## Metadata

| Query                                        | Value                                                                                      |
| -------------------------------------------- | ------------------------------------------------------------------------------------------ |
| Destination Branch                           | `develop`                                                                            |
| Design spec                                  |                                                                               |
| Change type                                  | ` Refactor Existing Code` <Pick one or more> |

### Should reviewers manually test your changes?

Yes

## Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my code.
- [ ] I unit tested new methods.
- [ ] I have generated the changeset file.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My changes require changes in documentation and I have made the corresponding changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Getting Started with Create React App
```
Comment (sachinsampath42) [inline README.md:1]: This heading should be a proper one

File: README.md [hunk 2]
```diff
@@ -17,54 +17,12 @@ You may also see any lint errors in the console.
...
 ### `npm run build`
 
 Builds the app for production to the `build` folder.\
 It correctly bundles React in production mode and optimizes the build for the best performance.
 
 The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
-
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
-
```
Comment (sachinsampath42) [inline README.md:31]: Keeping these scripts are not necessary.

Comment (SachinthaSampath) [reply README.md:31]: Better remove them next time.

**Review by sachinsampath42 — APPROVED** [review]
This should be good one for the start. Keep it up.

**Pull request merged**